erpnext/utilities/doctype/rename_tool/rename_tool.js

```diff
@@ -11,6 +11,17 @@ frappe.ui.form.on("Rename Tool", {
 			}
 		});
 	},
+	onload_post_render: function(frm) {
+		$(frm.get_field('file_to_rename').$input).click(() => {
+			frm.get_field('file_to_rename')
```

Comment:
better handled by setting df.option 😅 

will push the change.

Reply:
Noted, thank you